### PR TITLE
Better if grammar

### PR DESCRIPTION
**Pull request by matklad**
@jajakobyly please review.

### CODE REVIEW

File: src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt [hunk 1]
```diff
@@ -124,11 +123,14 @@ fun createSpacingBuilder(commonSettings: CommonCodeStyleSettings, rustSettings:
         .betweenInside(IDENTIFIER, EXCL, MACRO_INVOCATION).spaces(0)
         .betweenInside(MACRO_INVOCATION, IDENTIFIER, MACRO_DEFINITION).spaces(1)
         .betweenInside(IDENTIFIER, MACRO_ARG, MACRO_DEFINITION).spaces(1)
+        .before(ELSE_BRANCH).spacing(1, 1, 0, false, 0)
+        .betweenInside(ELSE, BLOCK, ELSE_BRANCH).spacing(1, 1, 0, false, 0)
```
Comment (jajakobyly) [inline src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt]: I wouldn't put this inside `//== macros` section but `//== expressions`

**Comment by jajakobyly** [comment]
lgtm :+1:

File: src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt (hunk 1 above)
Comment (matklad) [inline src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt]: :sweat_smile: I though I did exactly that

**Comment by matklad** [comment]
Merged manually

**Pull request closed without merging**